### PR TITLE
Admin

### PR DESCRIPTION
**Pull request by Vyshnav001**

### CODE REVIEW

**Comment by joywin2003** [comment]
there are merge conflicts, please fix them

**Comment by Vyshnav001** [comment]
> there are merge conflicts, please fix them

Done

File: src/middleware.ts [hunk 1]
```diff
@@ -5,17 +5,17 @@ import { getToken } from "next-auth/jwt";
...
+        }
     }
-  }
 }
 
 // See "Matching Paths" below to learn more
 export const config = {
-  matcher: ["/:path*", "/"],
+    matcher: ["/:path*", "/"],
 };
```
Comment (joywin2003) [inline src/middleware.ts:21]: this file didnt change at all, what did you do here lol

**Review by joywin2003 — APPROVED** [review]
Yea looks good

**Pull request merged**